frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditLayoutSetForSubform/EditLayoutSet/CreateNewSubformSection/SubformDataModel.tsx

```diff
@@ -55,12 +63,15 @@ export const SubformDataModel = ({
         )}
       </StudioNativeSelect>
       {displayDataModelInput ? (
-        <StudioTextfield
-          name='newSubformDataModel'
-          label={t('ux_editor.component_properties.subform.create_new_data_model_label')}
-          size='sm'
-          onChange={(e) => handleDataModel(e.target.value)}
-        />
+        <>
+          <StudioTextfield
+            name='newSubformDataModel'
+            label={t('ux_editor.component_properties.subform.create_new_data_model_label')}
+            size='sm'
+            onChange={(e) => handleDataModel(e.target.value)}
+          />
+          {nameError && <ErrorMessage>{nameError}</ErrorMessage>}
```

Comment:
```suggestion
            size='sm'
            onChange={(e) => handleDataModel(e.target.value)}
            error={dataModelNameError}
          />
```
In this case, instead of using the `<ErrorMessage>` from the designsystemet, send the error message in the prop of StudioTextField. Also perhaps change the name of `nameError` to `dataModelNameError`, and the same for the `nameError` for subform name. 
As you can see the styling is different otherwise:

![image](https://github.com/user-attachments/assets/a3c3ddac-b1fa-4232-ab63-223d0aa9f3ce)

Reply:
Fixed